src/pages/myProfile.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import Table from 'react-bootstrap/Table';
+
+const MyProfile = () => {
+  const rockets = useSelector((state) => state.rockets);
+  const missions = useSelector((state) => state.missions.missions);
+  const reservedRockets = rockets.filter((rocket) => rocket.reserved);
+  const joinedMissions = missions.filter((mission) => mission.reserved);
```

Comment:
- [ ] Hello you have done a great job on thi milestone ,i clone your project several time with different ideal but you page not display or render anything on screen ,after i check your code base i can see that you didnt import the myProfile in your myProfile.jsx .so this make the page nor render kindly fix it